### ISSUE TITLE
Pretrain BYOL and SwAV -WandB problem

### ISSUE THREAD

**Issue by kalikhademi** [vturrisi/solo-learn#364]
Hi,

I am using main_pretrain.py to train resnet 50 backbones with BYOL and SwAV method using unlabeled OpenImages dataset. After finishing the runs, SwAV wandb shows a chart for train_swav_loss_step but there is not such  chart for BYOL. I would like to have both of them. Could you please help me to make this change

**Comment by vturrisi** [comment]
Hey,
The losses for the two methods are different, so they are logged into separate plots. BYOL's loss is logged as `train_neg_cos_sim`.
